templates/kotsadm-config.yaml

```diff
@@ -9,6 +9,7 @@ data:
   minio-enabled-snapshots: "false"
   registry-is-read-only: "false"
   skip-preflights: {{ .Values.automation.skipPreflights | quote }}
+  with-minio: {{ .Values.withMinio | quote }}
```

Comment:
i don't like the fact that we'd be maintaining two permutations going forward. is there anything else that uses the kots helm chart other than embedded cluster today?

Reply:
not that i'm aware of in our product stack. i have seen multiple customers make PRs to this repo, so it could be in use by someone

Reply:
good question. i don't think so in the Replicated product stack today. there have been customers that use this for various things, but if it's not a supported path I could see us removing that. do you think that it should be a part of this PR?